common/on_actions/eaw_on_actions.txt

```diff
@@ -3799,188 +4287,2316 @@ on_actions = {
 								NOT = { country_exists = LSC }
 								FROM = { tag = LSM }
 							}
-						}
-					}
-					country_event = { id = longsword.78 days = 1 }
-				}
-				if = {
-					limit = {
-						tag = JER
-						has_completed_focus = JER_state_of_aquileia
-						OR = {
-							AND = {
-                                OR = {
-								    NOT = {country_exists = JRR }
-									JRR = { is_subject_of = JER }
-                                }
-                                OR = {
-								    NOT = { country_exists = VIN }
-									VIN = { is_subject_of = JER }
-                                }
-                                OR = {
-								    NOT = { country_exists = JFT }
-									JFT = { is_subject_of = JER }
-                                }
-								FROM = { tag = JEB }
+						}
+					}
```

Comment:
WHY!?

Reply:
Git incorrectly selects the brackets, nothing was removed.